flow/scripts/global_route.tcl

```diff
@@ -16,9 +16,16 @@ if {[info exist env(FASTROUTE_TCL)]} {
   }
 }
 
+# The default behavior if the user didn't specify GLOBAL_ROUTE_ARGS is to
+# produce a drc report every 5 iterations.
+#
+# If GLOBAL_ROUTE_ARGS is specified, then we do only what the
+# GLOBAL_ROUTE_ARGS specifies.
 global_route -guide_file $env(RESULTS_DIR)/route.guide \
                -congestion_report_file $env(REPORTS_DIR)/congestion.rpt \
-               {*}[expr {[info exists ::env(GLOBAL_ROUTE_ARGS)] ? $::env(GLOBAL_ROUTE_ARGS) : {-congestion_iterations 20 -verbose}}]
+               {*}[expr {[info exists ::env(GLOBAL_ROUTE_ARGS)] ? $::env(GLOBAL_ROUTE_ARGS) : \
+               {-congestion_iterations 20 -congestion_report_iter_step 5 -verbose}}]
```

Comment:
"-congestion_iterations 20 " is unrelated and I don't think belongs.

Reply:
It was already there, if you scroll right, you see it. I just added a linebreak as that line was getting long....